cypress/e2e/augury.cy.ts

```diff
@@ -57,4 +57,34 @@ describe("Augury", () => {
   it("creates flights and calculates inventory?", () => {
     // TODO
   });
+
+  it("creates and deletes creatives", () => {
+    const now = new Date().toISOString();
+    const canary = `Acceptance Test: ${now}`;
+    const audioFile = "cypress/samples/two-tone.mp3";
+
+    cy.visit("/creatives");
+    cy.get("h1").contains("Creatives");
+
+    // create creative
+    cy.get("a.btn-success[href='/creatives/new']").click();
+    cy.contains("New Creative");
+    cy.get("#creative_filename").type(`Creative ${canary}`);
+    cy.get(".uploader-container").get("input[type=file]").selectFile(audioFile, { force: true });
+    cy.get("#creative_account_id").next().type("{downArrow}{downArrow}{enter}");
+    cy.intercept("/options/advertisers?q= ").as("advertiserList");
+    cy.get("#creative_advertiser_id").next().type(" ");
+    cy.wait("@advertiserList");
+    cy.get("input:focus").type("{downArrow}{enter}");
+    cy.contains("Create Creative").click();
+    cy.contains("Creative was successfully created");
+
+    // delete creative
+    cy.visit("/creatives");
+    cy.contains(`Creative ${canary}`).click();
+    cy.get("#list-settings-list").next().children().first().click();
+    cy.contains("Delete Creative").click();
+    cy.get("button.btn-danger").contains("Delete Creative").click();
```

Comment:
I believe this also can be compacted to `cy.contains("button.btn-danger", "Delete Creative");`